### PR TITLE
Implement copy, cut and paste

### PR DESCRIPTION
**Pull request by tanbowensg**
#103

### CODE REVIEW

**Comment by Yuyz0112** [comment]
@Sczlog PTAL

File: packages/editor/src/operations/leaf/component/cutComponentLeafOperation.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import { ApplicationComponent } from '@sunmao-ui/core';
+import produce from 'immer';
+import { tryOriginal } from 'operations/util';
+import { BaseLeafOperation } from '../../type';
+
+export type CutComponentLeafOperationContext = {
+  componentId: string;
+};
+
+export class CutComponentLeafOperation extends BaseLeafOperation<CutComponentLeafOperationContext> {
```
Comment (Sczlog) [inline packages/editor/src/operations/leaf/component/cutComponentLeafOperation.ts:10]: we can reuse deleteCompoenentLeafOperation here.

**Review by Sczlog — CHANGES_REQUESTED** [review]
cut operation can be reused, others LGTM.

**Review by Sczlog — APPROVED** [review]
LGTM

**Comment by Yuyz0112** [comment]
@tanbowensg please resolve the conflicts.

**Pull request merged**